config.json

```diff
@@ -1419,15 +1419,52 @@
         "nugetId": "Xamarin.Google.Dagger",
         "dependencyOnly": true
       },
+      {
+        "groupId": "com.google.code.findbugs",
+        "artifactId": "jsr305",
+        "version": "3.0.1",
+        "nugetVersion": "3.0.1",
+        "nugetId": "Xamarin.Google.Code.FindBugs.JSR305",
+        "dependencyOnly": false,
+        "templateSet": "findbugs"      
+      },
       {
         "groupId": "org.chromium.net",
         "artifactId": "cronet-api",
-        "version": "76.3809.111",
-        "nugetVersion": "76.3809.111",
+        "version": "72.3626.96",
```

Comment:
The most recent NuGets for all of the Cronet's is `76.3809.111`.  Is there a reason bind an older version instead?